### ISSUE TITLE
when class only one prop not working

### ISSUE THREAD

**Issue by pedoc** [cezarypiatek/MappingGenerator#84]
sample code
```
public class Person
        {
            public int Age { get; set; }
            public string Name { get; set; }

            public Person(int age)
            {
            }
        }
```
but
```
 public Person(int age,string name)
            {
            }
```
it's working

env:
vs 2019 preview 3
win10 1903

**Comment by cezarypiatek** [comment]
@pedoc  here's a version with a fix for that issue, would you mind to test it? Any feedback is welcome.

https://ci.appveyor.com/api/buildjobs/85h1mcdvbmxv4xaw/artifacts/MappingGenerator%2FMappingGenerator%2FMappingGenerator.Vsix%2Fbin%2FRelease%2FMappingGenerator.vsix

**Comment by pedoc** [comment]
@cezarypiatek yes,it'working. thanks

**Comment by cezarypiatek** [comment]
Thanks. Fixed by #86
